app/src/app/api/v0/emissions-factor/route.ts

```diff
@@ -39,6 +64,39 @@ export const GET = apiHandler(async (req: NextRequest, _context: {}) => {
       whereClause.methodologyName = "modeled-data";
   }
 
+  if (!!metadata) {
+    let andCondition = [];
+    for (let key in metadata) {
+      let clause = {
+        [Op.or]: [
+          {
+            [`metadata.${key}`]: "nan",
```

Comment:
I think you can use the literal `NaN` instead of a string here? Or does that not work with the types?

Reply:
For this it needs to be the actual 'nan' string. that's the default string the mau created for factors that are applicable for all options